src/cargotype.cpp

```diff
@@ -47,6 +47,7 @@ void SetupCargoForClimate(LandscapeID l)
 
 		/* Set defaults for newer properties, which old GRFs do not know */
 		CargoSpec::Get(i)->multiplier = 0x100;
+		CargoSpec::Get(i)->cargo_station_age_period = 740; // 10 days.
```

Comment:
Consider using DAY_TICKS here?

Reply:
Well, it's actually 4 * def cargo ageing factor (which happens to be 10 days).